### PR TITLE
tests: stop restarting journald service on prepare

### PR DESCRIPTION
**Pull request by sergiocazzolato**
It was found the root cause for the issue that was forcing to restart the journald service. 

The issue is tha the network-bind-consumer snap in the interfaces-network-bind test is creating thousands of DENIALS in the journal. Most of those are sent after the journalctl cursor for following test is determined producing errors while preparing the test (checking is the journal is ready).

The solultion is to remove the snap on the restore. 

To see the error: https://travis-ci.org/snapcore/snapd/builds/400137393?utm_source=github_status&utm_medium=notification

### CODE REVIEW

**Comment by sergiocazzolato** [comment]
Please, dont merge this until we have many greens on travis

File: tests/main/interfaces-network-bind/task.yaml [hunk 1]
```diff
@@ -31,6 +31,12 @@ prepare: |
 
     EOF
 
+restore: |
+    # This snap is removed because it generates thousands of DENIALS in the journal. Most of those
+    # are sent after the journalctl cursor for following test is determined producing errors while
+    # preparing the test.
+    snap remove network-bind-consumer
```
Comment (bboozzoo) [inline tests/main/interfaces-network-bind/task.yaml]: Wondering if other tests leave potentially problematic snaps behind.

Comment (sergiocazzolato) [reply tests/main/interfaces-network-bind/task.yaml]: @bboozzoo, I just saw this one, in fact the same snap in other tests is not causing problems like this.

**Comment by sergiocazzolato** [comment]
@mvo5 I already executed 8 times this PR and the error is not reproduced anymore. Do you think we can land it?

**Comment by mvo5** [comment]
@sergiocazzolato Sure, I just set it to blocked to avoid accidental merging.

**Pull request merged**